cms/templates/export_git.html

```diff
@@ -49,9 +49,9 @@ <h2 class="title">${_("Export Course to Git:")}</h2>
       <div class="messages">
         % if msg:
           % if failed:
-          <h3 class="error-text">${_('Export Failed')}:</h3>
+          <h3 class="error-text">${_('Export Failed:')}</h3>
```

Comment:
Ah wait I see, ti's there just outside the string.

I assume that other languages don't use the colon?